examples/components/http-server-with-hono/README.md

```diff
@@ -35,10 +35,7 @@ The adapter in [`src/wasmcloud/hono-adapter`](https://github.com/wasmCloud/types
 
 **How it works:**
```

Comment:
May be a separate discussion but curious about the need to check-in generated wit and types.
The `wash dev`  command will already fetch everything in `wit/deps` and the `build` script is generating types.

A quick look and seems like other examples skipped checking in deps but all are checking in types (probably because they don't use `jco guest-types` yet)

Reply:
Yeah, I agree. I have a backlog item to make a few more simplifications like that. Neither the types, nor the deps really need to be checked in.